### PR TITLE
Bugfix/wiki runner step description

### PR DESCRIPTION
**Pull request by EduardKrieger**

### CODE REVIEW

File: documentation/Functions.md [hunk 1]
```diff
@@ -349,20 +349,46 @@ path: The URL path on which is checked if the server is running
 If the tutorial should be tested on the console environment, you have to specify a port.
 ***
 
-### nextKatacodaStep <a name="nextKatacodaStep"></a>
+### displayContent <a name="displayContent"></a>
 #### parameter
-1. The title of the step.
-2. An array of json objects with files, content, or images to be rendered within the katacoda step.
+1. The title of the step. 
+2. An array of json objects with files, content, or images to be rendered within the katacoda step. The use for this function is to display an image and some descriptive text. No Katacoda Syntax is allowed in the files or the content!
```
Comment (GuentherJulian) [inline documentation/Functions.md:355]: Katacoda with capital first letter

File: documentation/Functions.md [hunk 2]
```diff
@@ -349,20 +349,46 @@ path: The URL path on which is checked if the server is running
...
-nextKatacodaStep("Step title", [{ "file": "files/description.md" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
+display("Step title", [{ "file": "files/description.asciidoc" }, { "content": "This is just plain content." }, { "image": "files/image.png" }])
 
 #### Details
 Available attributes in the json objects:
 
-file: Path to a file whose content is to be displayed in the katacoda step (e.g. .md or .txt file).
-content: Plain text to be displayed in the katacoda step.
-image: Path to an image to be displayed in the katacoda step.
+1. file: Path to a file whose content is to be displayed in the katacoda step (e.g. .asciidoc or .txt file). The file should be following the formating of asciidoc files.
```
Comment (GuentherJulian) [inline documentation/Functions.md:363]: Katacoda with capital first letter. Check also in the rest of the text
formatting

File: documentation/Functions.md [hunk 3]
```diff
@@ -349,20 +349,46 @@ path: The URL path on which is checked if the server is running
...
 
-file: Path to a file whose content is to be displayed in the katacoda step (e.g. .md or .txt file).
-content: Plain text to be displayed in the katacoda step.
-image: Path to an image to be displayed in the katacoda step.
+1. file: Path to a file whose content is to be displayed in the katacoda step (e.g. .asciidoc or .txt file). The file should be following the formating of asciidoc files. 
+2. content: Plain text to be displayed in the katacoda step. This Text should be following the formating of asciidoc files.
+3. image: Path to an image to be displayed in the katacoda step.
+
+#### Formatting rules for content and .asciidoc or .txt files.
+* You can add headers to structure your text. The generated headers are shown in the examples below. The headers should fit into the overall structure of the generated wiki so level 1 header arent allowed, but the other header can be used at your judgement.
```
Comment (GuentherJulian) [inline documentation/Functions.md:368]: aren't

File: engine/wikiRunner.ts [hunk 4]
```diff
@@ -11,13 +12,17 @@ export abstract class WikiRunner extends Runner {
     [".js", "javascript"], [".html", "html"],
     [".scss", "css"], [".asciidoc", "asciidoc"], ]);
     protected readonly INSTALLED_TOOLS: string = "installedTools";
+    public CommandCntMap = new Map();
```
Comment (GuentherJulian) [inline engine/wikiRunner.ts:15]: Start variable names with lower case letters

File: runners/wikiConsole/templates/addSetupScript.asciidoc [hunk 5]
```diff
@@ -1,11 +1,15 @@
-== Run a Setupscript
-Run a Script to setup the enviroment for the Tutorial.  
+<%if(title){%>== <%- title; %>
+<%if(text){%><%- text; %><%}%><%}else{%>
+
+=== Run a Setupscript
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/addSetupScript.asciidoc:4]: === Run a setup script

File: runners/wikiConsole/templates/addSetupScript.asciidoc [hunk 6]
```diff
@@ -1,11 +1,15 @@
...
+<%if(text){%><%- text; %><%}%><%}%>
 
+Run a Script to setup the enviroment for the Tutorial.
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/addSetupScript.asciidoc:7]: script

File: runners/wikiConsole/templates/addSetupScript.asciidoc [hunk 7]
```diff
@@ -1,11 +1,15 @@
...
+<%if(text){%><%- text; %><%}%><%}%>
 
+Run a Script to setup the enviroment for the Tutorial.  
 
-=== Prerequisites
+==== Prerequisites
 * Any Editor that can edit files.
 
-=== Create the Setupscript
+==== Create the Setupscript
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/addSetupScript.asciidoc:12]: setup script

File: runners/wikiConsole/templates/addSetupScript.asciidoc [hunk 8]
```diff
@@ -29,11 +33,13 @@ The editor opens a new editor window for an untitled file where you can insert t
 Save the file by selecting *Save* or *Save as* in the file context menu or by using the keyboard shortcut ctrl+s.
 A file explorer window opens where you should set the filename to *<%= scriptNameLinux; %>* and save it to your current directory. 
 
-=== Run the Setupscript
+==== Run the Setupscript
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/addSetupScript.asciidoc:36]: setup script

File: runners/wikiConsole/templates/createFolder.asciidoc [hunk 9]
```diff
@@ -1,15 +1,18 @@
...
-=============
+<%if(title){%>== <%- title; %>
+<%if(text){%><%- text; %><%}%><%}else{%>
 
+=== Create Folder
+<%if(text){%><%- text; %><%}%><%}%>
 
-Creating a new Folder
-~~~~~~~~~~~~~~~~~~~~~
+==== Creating a new Folder
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createFolder.asciidoc:7]: folder

File: runners/wikiConsole/templates/createFolder.asciidoc [hunk 10]
```diff
@@ -1,15 +1,18 @@
-Create Folder
-=============
+<%if(title){%>== <%- title; %>
+<%if(text){%><%- text; %><%}%><%}else{%>
 
+=== Create Folder
```
Comment (GuentherJulian) [inline runners/wikiConsole/templates/createFolder.asciidoc:4]: folder

File: runners/wikiEclipse/templates/adaptTemplates.asciidoc [hunk 11]
```diff
@@ -1,11 +1,17 @@
-== Generate new Templates for CobiGen 
+<%if(title){%>== <%- title; %>
+<%if(text){%><%- text; %><%}%><%}else{%>
+
+=== Generate new Templates for CobiGen
```
Comment (GuentherJulian) [inline runners/wikiEclipse/templates/adaptTemplates.asciidoc:4]: templates

File: runners/wikiEclipse/templates/adaptTemplates.asciidoc [hunk 12]
```diff
@@ -1,11 +1,17 @@
...
+
 How to use CobiGen, to generate templates for the following code generation.
 More information about CobiGen on https://devonfw.com/website/pages/docs/master-cobigen.asciidoc.html.
 
-=== Prerequisites
+==== Prerequisites
 An installed devonfw distribution is needed. To execute CobiGen in Eclipse IDE, you have to install the devonfw IDE with the Eclipse IDE first. Follow the https://devonfw.com/website/pages/docs/devonfw-ide-introduction.asciidoc.html[devonfw-ide] documentation to install the same.
 
-=== Generate CobiGen Templates with Eclipse IDE
+==== Generate CobiGen Templates with Eclipse IDE
```
Comment (GuentherJulian) [inline runners/wikiEclipse/templates/adaptTemplates.asciidoc:14]: templates

**Review by GuentherJulian — CHANGES_REQUESTED** [review]

File: documentation/Functions.md (hunk 1 above)
Comment (EduardKrieger) [reply documentation/Functions.md:355]: Changed it lower case

File: documentation/Functions.md (hunk 2 above)
Comment (EduardKrieger) [reply documentation/Functions.md:363]: Changed it

File: documentation/Functions.md (hunk 3 above)
Comment (EduardKrieger) [reply documentation/Functions.md:368]: Used are not in this case

File: engine/wikiRunner.ts (hunk 4 above)
Comment (EduardKrieger) [reply engine/wikiRunner.ts:15]: Changed it to lower case

File: runners/wikiConsole/templates/addSetupScript.asciidoc (hunk 5 above)
Comment (EduardKrieger) [reply runners/wikiConsole/templates/addSetupScript.asciidoc:4]: Changed it

File: runners/wikiConsole/templates/addSetupScript.asciidoc (hunk 8 above)
Comment (EduardKrieger) [reply runners/wikiConsole/templates/addSetupScript.asciidoc:36]: Changed it

File: runners/wikiConsole/templates/createFolder.asciidoc (hunk 9 above)
Comment (EduardKrieger) [reply runners/wikiConsole/templates/createFolder.asciidoc:7]: Changed it

File: runners/wikiConsole/templates/createFolder.asciidoc (hunk 10 above)
Comment (EduardKrieger) [reply runners/wikiConsole/templates/createFolder.asciidoc:4]: Changed it

File: runners/wikiEclipse/templates/adaptTemplates.asciidoc (hunk 11 above)
Comment (EduardKrieger) [reply runners/wikiEclipse/templates/adaptTemplates.asciidoc:4]: Changed it

File: runners/wikiEclipse/templates/adaptTemplates.asciidoc (hunk 12 above)
Comment (EduardKrieger) [reply runners/wikiEclipse/templates/adaptTemplates.asciidoc:14]: Changed it

**Review by GuentherJulian — APPROVED** [review]

**Pull request merged**